app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,5 +1,5 @@
-<% if presenter.representative_id.present? && (fs = ::FileSet.find(presenter.representative_id)) %>
-  <%= media_display fs %>
+<% if presenter.representative_id.present? %>
```

Comment:
This if statement and the `return nil if representative_id.blank?` of `#representative_presenter` are a duplication of knowledge. I'm wondering if removing the if statement and relying on the media_display method in the helper to ask the questions.

It would make testing the view logic easier (as we push it to the helper).

So in `_representative_media.html.erb`
```erb
<%= media_display(presenter.representative_presenter) %>
```

Then in `file_set_helper.rb`

```ruby
def media_display(presenter, locals = {})
  return image_tag('nope.png', class: "canonical-image") unless presenter.present?
  render(media_display_partial(presenter), locals.merge(file_set: presenter))
end
```